src/main/java/org/photonvision/PhotonCamera.java

```diff
@@ -105,6 +109,30 @@ public void setDriverMode(boolean driverMode) {
     }
   }
 
+  /**
+   * Changes commands to store single-frame captures to memory.
+   * Commands should be set to true if capturing an image is desired.
+   * When the camera sees a command transition from false to true, it will save the next camera frame to disk.
+   * Commands are automatically reset to false after 500ms.
+   * Images take up space in the filesystem of the PhotonCamera - calling it frequently will fill up
+   * disk space and eventually cause the system to stop working. 
+   * Clear out images in /opt/photonvision/photonvision_config/imgSaves frequently to prevent issues.
+   * @param inputImgCaptureCmd Triggers capture of the input image stream
+   * @param outputImgCaptureCmd Triggers capture of the output image stream
+   */
+  public void setImageCaptureCmds(boolean inputImgCaptureCmd, boolean outputImgCaptureCmd) {
```

Comment:
Would it be clearer if this were an overload (i.e. named `captureImages`)? Seems like it's less confusing to say "capture some images" instead of saying "set an image capture command," especially when there's another very similar method.

Reply:
I'm down with renaming - IMO as long as the functionality is there, I'd default to the maintainers to define strategy and naming to ensure end-user experience is a good one.

I'd chosen the names sort-of intentionally with the assumption the two would get used differently:

To get an image to be saved to disk, the user has to generate rising edges on the NT boolean command entry. I'm assuming we don't want these API's to block, meaning we're pushing it to the user to flip them false/true (or wait the 500ms for the camera to reset the command to false). 

The rough intent was that `setImageCaptureCmds` assumes the user will manually set each to true and false in periodic tasking to accomplish those rising edges, while `captureImages` assumes they'll set it once and wait the 500ms for the camera to reset it.

One other way to resolve.... which maybe @Bankst or @mcm001  could chime in on - would we maybe want to change the boolean NT command to an integer? Had the idea today - RIO side would increment the integer, camera side would see the number change and take a new picture. It would be less intuitive if you were looking at the NT entries, but could allow us to do a non-blocking single `captureImages()` API in photonlib which guarantees one image set per call. Such a change [would also impact the other PR](https://github.com/PhotonVision/photonvision/pull/134), but it would be pretty straightforward.